### DISCUSSION TITLE
Quodlibet metadata location

### DISCUSSION THREAD

**Discussion by lot785** [quodlibet/quodlibet#4347] [Q&A] [answered]
For the life of me I cannot locate Quodlibet's metadata on my Mac.  I hate to spend a lot of time creating tags when I cannot backup.  Any help on locations?  The Q&A was no help.  Thanks.

**Comment by bassstorm** [comment, marked as answer]
Hi,

It must be `~/.quodlibet`.
See more details at https://quodlibet.readthedocs.io/en/latest/guide/faq.html#metadatalocation.
